tests/WP_Ultimo/Managers/Email_Manager_Test.php

```
@@ -0,0 +1,151 @@
+		$count_before = count($emails_before);
+
+		// Try to create the same system email twice
+		$email_data = [
+			'slug'    => 'payment_received_admin',
+			'title'   => 'Test Payment Email',
+			'content' => 'Test content',
+			'event'   => 'payment_received',
+			'target'  => 'admin',
+		];
+
+		$result1 = $this->manager->create_system_email($email_data);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result1'. (undefined)

(UnusedLocalVariable)